chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/CertifiedProductDetailsDTO.java

```diff
@@ -240,6 +250,46 @@ public void setDecertificationDate(Date decertificationDate) {
         this.decertificationDate = Util.getNewDate(decertificationDate);
     }
 
+    public String getRwtPlanUrl() {
+        return rwtPlanUrl;
+    }
+
+    public void setRwtPlanUrl(String rwtPlanUrl) {
+        this.rwtPlanUrl = rwtPlanUrl;
```

Comment:
Do you need getters/setters for the String fields? This class is annotated with @Data I think.